bpf/kmesh/ads/include/ctx/sock_addr.h

```diff
@@ -10,6 +10,9 @@ typedef struct bpf_sock_addr ctx_buff_t;
 #define KMESH_PORG_CALLS    cgroup/connect4
 // clang-format on
 
+// tail_call map dont support pinning when shared by different bpf types, so define different name in sockops & sockconn
+#define map_of_tail_call_prog km_cgrptailcall
```

Comment:
For other reviewers, this is making the map named `km_cgrptailcall` in sock_addr prog

And below make anothe map `km_skopstailcall` in sock_ops prog. 

So the code can be reused `kmesh_tail_call` by different progs without passing in the map name